doc/reference/tooling/tcm/tcm_releases/tcm_1.1.rst

```diff
@@ -0,0 +1,117 @@
+.. _tcm_releases_1_1:
+
+Tarantool Cluster Manager 1.1
+=============================
+
+Release date: May 16, 2024
+
+Releases in series: 1.1.0
+
+|tcm_full_name| 1.1 introduces a number of new features that extend and improve its
+cluster management capabilities. Below is an overview of its key updates.
+
+.. _tcm_releases_1_1_data_access:
+
+Data access
+-----------
+
+An important update of |tcm| 1.1.0 is a set of features that enable access to clusters'
```

Comment:
> 1.1.0

A short version (1.1) might look better in a what's new text (as in the what's new intro).